package.json

```diff
@@ -32,6 +32,7 @@
     "mBaaS"
   ],
   "scripts": {
+    "lint": "eslint .",
```

Comment:
For long term, I think we should remove `eslint` config from `Gruntfile` and use the usual posttest + lint scripts, as we do in other projects. However, that's a more complex change that should have its own pull request.

I am ok to add `"lint": "grunt eslint"` as a quick fix though.